therapy/etl/wikidata.py

```diff
@@ -28,6 +27,13 @@
     "wikidata": NamespacePrefix.WIKIDATA.value,
 }
 
+# Provide standard concept ID prefixes
+ID_PREFIXES = {
+    "wikidata": "Q",
+    "chembl": "CHEMBL",
+    "drugbank": "DB",
+    "ncit": "C",
+}
```

Comment:
This is in `SourceIDAfterNamespace`. Maybe use it instead?

Reply:
Oh, I meant to pose this as a question and forgot -- `SourceIDAfterNamespace` doesn't get used anywhere outside of this line anymore, so I was thinking about moving it out of schemas and into here (like so). What're your thoughts?

Reply:
Ahh. If this is the only place we use it, we can leave it here for now. If we end up using it in other places, then we can move it to schemas then.